ledger/accountdb.go

```diff
@@ -2002,26 +2002,28 @@ func (qs *accountsDbQueries) lookupAllResources(addr basics.Address) (data []per
 		}
 		defer rows.Close()
 
-		var addrid int64
+		var addrid, aidx, rtype sql.NullInt64
 		var dbRound basics.Round
-		var aidx basics.CreatableIndex
-		var rtype basics.CreatableType
 		data = nil
 		var buf []byte
 		for rows.Next() {
 			err := rows.Scan(&addrid, &dbRound, &aidx, &rtype, &buf)
 			if err != nil {
 				return err
 			}
+			if !addrid.Valid || !aidx.Valid || !rtype.Valid {
```

Comment:
how do we know it is the first entry?

Reply:
oh right, well I just copied that logic from accountsDbQueries.listCreatables. Since addrid/aidx/rtype are NOT NULL columns you would really only ever get this in the empty case, but I see your point..

Reply:
You fixed it below.